sdks/python/apache_beam/runners/portability/local_job_service_main.py

```diff
@@ -99,11 +104,23 @@ def run(argv):
       options.port_file = os.path.splitext(options.pid_file)[0] + '.port'
       argv.append('--port_file')
       argv.append(options.port_file)
+
+    if not options.stdout_file:
+      raise RuntimeError('--stdout_file must be specified with --background')
+    stdout_dest = open(options.stdout_file, mode='w')
```

Comment:
I think I mentioned this before, but is it an issue that these file descriptors might get closed on completion of this process?

Reply:
Yea I think my rebasing clobbered that thread. It is not an issue. Parent file descriptors are not closed. You can find some links I think on the PR front page it will still have the prior conversation.

(I won't rebase from here on out, until review is done)

Reply:
No problem. My comment was from something like a mont ago. Thanks for the references.